src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +203,30 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      var urlChanged = false;
+      controller = this.controller;
+
+      if (currentUrl.replace(/[\?&]_=[^&]+$/,'') != url)
+      {
+        currentUrl = url || "";
+
+        if (currentUrl.length > 0) 
+        {
+          currentUrl = currentUrl + (/\?/.test(currentUrl) ? "&" : "?") + "_=" + getRandomString();
+        }
+        urlChanged = true;
+        loaded = false;
+        url = "setVideoUrl("+currentUrl+")";
+      }
+      if (_.isEmpty(currentUrl)) 
+      {
+        controller.notify(controller.EVENTS.ERROR, { errorcode: 0 }); 
```

Comment:
We just save the value of URL on flash side and we load the same when load() is called.  In this scenario, you mean to say call flash to save the value as empty string and then VTC will call load to handle unload() or do we need to call destroy().